### DISCUSSION TITLE
想让分享的页面展示为编辑时的样式效果，如何设置？

### DISCUSSION THREAD

**Discussion by kkk-gitbook** [GitbookIO/community#640] [2. Published content]
![image](https://github.com/GitbookIO/community/assets/172463681/bc59ed01-d464-4add-b4dc-a5d63b89ae22)

https://shu.gitbook.io/

**Comment by aikoii** [comment]
Hey @kkk-gitbook, thanks for reaching out. We provide support only in English, I hope you don't mind 😊. A similar question was asked and answered here: [GitBook Discussion](https://github.com/orgs/GitbookIO/discussions/595#discussioncomment-9232201).

I'll close this conversation to keep things streamlined and encourage you to vote or comment on the previous post. Thanks!
